apps/nextjs-website/src/app/[productSlug]/quick-start/page.tsx

```diff
@@ -22,6 +24,28 @@ export type QuickStartGuidePageProps = {
   readonly steps?: ReadonlyArray<Step>;
 } & LayoutProps;
 
+export async function generateMetadata(
+  { params }: ProductParams,
+  parent: ResolvingMetadata
+): Promise<Metadata> {
+  const { shared } = translations;
+  const previousTitle = (await parent).title || shared.siteTitle;
+  const { abstract, path, product } = await getQuickStartGuide(
+    params?.productSlug
+  );
+  const title = `${previousTitle} - ${abstract?.title}`;
```

Comment:
What's happen if the `abstract` is `null` or `undefined`? The title would be something like `Hello - `?

Assume the value of `previousTitle` variable is `Hello`